app/src/main/java/ch/epfl/sweng/SDP/game/drawing/Item.java

```diff
@@ -0,0 +1,37 @@
+package ch.epfl.sweng.SDP.game.drawing;
+
+public abstract class Item {
+
+    protected int x;
+    protected int y;
+    protected int radius;
+    protected int interval;
+
+    protected Item(int x, int y, int radius, int interval) {
+        if(x < 0 || y < 0 || radius < 0 || interval < 0) {
```

Comment:
There is a static method checkPrecondition inside Preconditions.java for doing this

Reply:
Thanks for pointing that out.